Rockerfile.mesa

```diff
@@ -29,31 +31,73 @@ FROM {{ $image }}:llvm-{{ .LLVM }}
 
 LABEL maintainer "Juan A. Suarez Romero <jasuarez@igalia.com>"
 
-RUN apt-get update                                   \
-  && apt-get -y --no-install-recommends install git  \
-     {{ if eq .BUILD "windows" }}mingw-w64{{ end }}  \
+RUN apt-get update                                      \
+  && apt-get -y --no-install-recommends install git     \
+     {{ if eq .BUILD "windows" }}mingw-w64{{ end }}     \
+     {{ if eq .BUILD "android" }}                       \
+       wget openjdk-8-jdk python sudo make gcc gettext  \
+       python-mako bison libc6-dev xz-utils             \
```

Comment:
Not really. For Android we are using as base system Ubuntu Xenial, not our base. Reason is that base contains a lot of dependencies that are not useful at lot for Android, and thus we can reduce the final size.